mods/ra/sequences/misc.yaml

```diff
@@ -128,6 +128,9 @@ pips:
 		Tick: 300
 		Offset: 0, -6
 
+disguise:
```

Comment:
Please define this as `tag-spy: tag-spy` after `tag-primary` inside `pips` and rename the file to `tag-spy.shp`. There's no reason to break convention with the fake and primary tags here.